Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,32 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+function getCardValue(cardString) {
+  cardStringPadded = cardString.padStart(3, "0");
+  rank = cardStringPadded.substring(0, 2);
+  if (2 <= rank && rank <= 10) return rank;
+  if (rank == "0A") return 11;
+  if (rank == "0Q" || rank == "0J" || rank == "0K") return 10;
```

Comment:
if cardstrin is "Q♠" , it will return "10" which is expected.  
with the input of "007♠" or "0Q♠" or "02Q♠" or"00Q♠" it should return: "invalid card rank". I wrote some assertion which check them . but it doesn't work as expected.
so I inserted these codes:const validRanks = [
    "2",
    "3",
    "4",
    "5",
    "6",
    "7",
    "8",
    "9",
    "10",
    "J",
    "Q",
    "K",
    "A",
  ];

  // Validate rank
  if (!validRanks.includes(rank)) {
    return "Invalid card rank";
  }
   .
   Now the assertions from line 81-98 are working properly!
Thanks CJ!